src/components/TopBar/TopBar.scss

```diff
@@ -68,7 +68,7 @@ $context-control-expand-after: 1400px;
   padding: spacing(tight);
   border-radius: border-radius();
   fill: var(--p-icon); // Icon will inherit this fill
-  transition: (duration() - 33ms) fill easing() 33ms;
+  transition: var(--p-duration-150) fill easing() 33ms;
```

Comment:
I was thinking we'd need to address the delay or whatever is at the end as well. We could add another token and do:

```suggestion
  transition: var(--p-duration-150) fill easing() var(--p-duration-50);
```

Reply:
I think we should add `ms` as a fail to polaris coverage and address in a follow up.